### ISSUE TITLE
NC 18.0.0.10 (from download after Nextcloud HUB annouce). 

### ISSUE THREAD

**Issue by noci2012** [nextcloud/server#18974]
After installing Nextcloud 18 from the .zip downloaded from the Nextcloud HUB site.
* Only office is non-functional.
* System fails completely after installing / enabling Push.
  Interface 'OCP\\Push\\Helper\\IPushHelper' not found at ...../apps/push/lib/Helper/PushHelper.php#78

**Comment by TheMrApostel** [comment]
@noci2012 My Onlyoffice is still functional after installing Nextcloud 18 from the HUB site.
After the upgrade via terminal the app was listed as "Disabled incompatible app: onlyoffice".
Did you reactivate the app?

**Comment by tflidd** [comment]
For push also consider posting in the push-repository:
https://github.com/nextcloud/push/issues

**Comment by markusschweitzer** [comment]
My onlyoffice was working in version 17. After the update to version 18, the menus for creating new office files are not there and office documents are downloaded instead of opened in onlyoffice. 
The document server is running (and reachable), the app is activated and the service address, secret and internal service address are all set correctly. In nextcloud.log there is nothing with "office".

Anything I can do?

**Comment by noci2012** [comment]
@TheMrApostel : I had NO Only office.. The video of the Nextcloud HUB release talked about downloading NextcloudHub V18.0 with a lot of functions activated and working from the start. Also mentioned onlyoffice as integrated solution available solution. 
So i Installed THAT version of Nextcloud from the announcement (to try out)  to find the 2 major issues mentioned here.
Documents are downloaded, not edited.

Note this was NO upgrade, new install.  (most important.. it doesn't work as advertised in a huge press conference.).
(For production use i have Collabora working, V17).

**Comment by noci2012** [comment]
@tflidd: https://github.com/nextcloud/push/issues/76

**Comment by TheMrApostel** [comment]
First, make sure that you have installed the apps through your installation
![image](https://user-images.githubusercontent.com/10365495/72829046-3a4ea580-3c7e-11ea-8a29-882e26e176b0.png)

But it still seems that you need to take some extra steps.
That is what worked for a fresh installation:

`wget https://github.com/nextcloud/documentserver_community/releases/download/v0.1.2/documentserver_community.tar.gz

tar -zxvf documentserver_community.tar.gz

mv documentserver_community  /your/path/to/nextcloud/apps

cd /var/www/nextcloud/apps

chown -R www-data.www-data documentserver_community`

**Comment by TheMrApostel** [comment]
> My onlyoffice was working in version 17.
> After the update to version 18, the menus for creating new office files are not there and office documents are downloaded instead of opened in onlyoffice.
> 
> The document server is running (and reachable), the app is activated and the service address, secret and internal service address are all set correctly. In nextcloud.log there is nothing with "office".
> 
> Anything I can do?
> 
> EDIT: I even updated the onlyoffice docker container to the latest version => no change

After the update to 18.0.0.10 I had to activate Onlyoffice by hand.
So far I haven't run into any problems

![image](https://user-images.githubusercontent.com/10365495/72829522-3a02da00-3c7f-11ea-8da9-d099a4c6f747.png)

**Comment by markusschweitzer** [comment]
For me it was: https://github.com/nextcloud/files_fulltextsearch/issues/83

TLDR: if you have the files_fulltextsearch app enabled, onlyoffice doesnt work :/

**Comment by noci2012** [comment]
> But it still seems that you need to take some extra steps.
> That is what worked for a fresh installation:
Well a small difference from the presentation.

**Comment by noci2012** [comment]
Closed, NC 18 not used anymore. No solution for only office.
Went with Collabora instead.
